### DISCUSSION TITLE
trivy.db database format?

### DISCUSSION THREAD

**Discussion by espeyrob** [aquasecurity/trivy#6519] [Q&A] [answered]
I'm trying to add my own custom CVE into my instance of trivy.db, but I've been unable to interact with the database because it's not in any format that I recognize. Does anyone know how I could modify it?

**Comment by simar7** [comment, marked as answer]
It's BoltDB https://github.com/boltdb/bolt#opening-a-database you can take a peek at it via something like https://github.com/Moniseeta/boltwiz

**Reply by knqyf263** [reply]
Also, more information here.
https://github.com/aquasecurity/trivy-db
